src/main/webapp/js/feedbackSubmissionsEdit.es6

```diff
@@ -904,6 +906,25 @@ function showModalWarningIfSessionClosed() {
     }
 }
 
+function showModalWarningMessageIfSessionIsClosig() {
+    const endDate = $('#endTime').text().split(' ');
+    const currentDate = new Date();
+    const locale = 'en-us';
+    const month = currentDate.toLocaleString(locale, {
+        month: 'short',
+    });
+
+    if (endDate[1] === currentDate.getDate() && endDate[2] === month
+            && endDate[3] === currentDate.getFullYear()) {
+        if (endDate[5] === currentDate.split(' ')[5]
+                && (endDate[4].split(':')[0] - currentDate[4].split(':')[0]) <= 15) {
```

Comment:
It would be better to first parse the text in `#endTime` into a date object, then use its methods `getDate()`, `getMonth()` etc.